client/main/backstage.js

```diff
@@ -43,7 +43,8 @@ const FSEntryListModel = Backbone.Model.extend({
         browseable: true,
         extensions: true,
         multiselect: false,
-        wdType: 'main'
+        wdType: 'main',
+        retrieving: true
```

Comment:
would it be more elegant to have a `status` property?

'ok', 'loading', 'error'

Reply:
Yes this is much better.